src/System.Text.CaseFolding/System/Text/CaseFolding/SimpleCaseFoldingMapBelow5FF.cs

```diff
@@ -0,0 +1,114 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Text.CaseFolding
+{
+    /// <summary>
+    /// Simple case folding methods.
+    /// </summary>
+    public static partial class SimpleCaseFolding
+
+    {
+        private static readonly char[] MapBelow5FF =
```

Comment:
>MapBelow5FF [](start = 39, length = 11)

was really worse it to special case this and adding 3K of data for it?

Reply:
Do you ask about 2 vs 3 level mapping?
Yes, 2 level mapping extremally fastest. This code is very sensitive and any change noticeably affects the speed.
This code is as fast as optimized code (chars < 0x7ff) for OrdinalIgnoreCase (!) on .Net Core 2.1 runtime and  noticeably faster (~7% !!!) on .Net Core 3.0 runtime.

Currently .Net Core is always optimized only for limited char set < 0x7ff and twice (!) slow for chars from other languages. __Why do applications have to be twice as slow for any EU?__ The proposed code eliminates this discrimination for most languages ​​of the EU and the Middle East.

Reply:
I am ok as long as you are seeing we are noticing good performance. I was trying to think in a way compact the size of the static data.

---
In reply to: [250613621](https://github.com/dotnet/corefxlab/pull/2637#discussion_r250613621) [](ancestors = 250613621)